OCR/ocr/services/image_ocr.py

```diff
@@ -1,30 +1,141 @@
+from collections.abc import Iterator
+
 from transformers import TrOCRProcessor, VisionEncoderDecoderModel
 import torch
 import numpy as np
+import cv2 as cv
 
 
 class ImageOCR:
     def __init__(self, model="microsoft/trocr-large-printed"):
         self.processor = TrOCRProcessor.from_pretrained(model)
         self.model = VisionEncoderDecoderModel.from_pretrained(model)
 
+    @staticmethod
+    def compute_line_angle(lines: list) -> Iterator[float]:
```

Comment:
Is there a max angle this likely starts breaking at? Like, should we handle a case where the `line_angle` is over 45 degrees?